lib/bag/item.rb

```diff
@@ -23,6 +23,20 @@ def item_id
     object_properties['objectId'].first.tr('/', '-')
   end
 
+  def owner_id
+    object_properties['ownerId']
+  end
+
+  def other_ids
+    object_properties['otherIds'].split(',')
+  end
+
+  def collection_handles
+    arr = Array.new
```

Comment:
This can be simplified, and referencing the two fields `other_ids` and `owner_id` .. `owner_id` is a single value so we shove it into an array and use `+` to combine the two.

```
[owner_id] + other_ids
```